### ISSUE TITLE
AWS EU-Central Image No Graylog Servers running

### ISSUE THREAD

**Issue by zeacom** [Graylog2/graylog2-images#46]
After running `ubuntu@graylog:~$ sudo graylog-ctl reconfigure` there is no process which listen on port 12900:

```
ubuntu@graylog:~$ sudo netstat -tap | grep LISTEN
tcp        0      0 *:57186                 *:*                     LISTEN      3115/java
tcp        0      0 *:3333                  *:*                     LISTEN      3115/java
tcp        0      0 *:27017                 *:*                     LISTEN      3141/mongod
tcp        0      0 localhost:2380          *:*                     LISTEN      3038/etcd
tcp        0      0 graylog:9200            *:*                     LISTEN      3115/java
tcp        0      0 *:http                  *:*                     LISTEN      3156/nginx.conf
tcp        0      0 *:38161                 *:*                     LISTEN      3115/java
tcp        0      0 graylog:9300            *:*                     LISTEN      3115/java
tcp        0      0 *:ssh                   *:*                     LISTEN      1081/sshd
tcp        0      0 localhost:afs3-callback *:*                     LISTEN      3038/etcd
tcp        0      0 *:https                 *:*                     LISTEN      3156/nginx.conf
tcp6       0      0 [::]:9000               [::]:*                  LISTEN      3070/java
tcp6       0      0 [::]:2379               [::]:*                  LISTEN      3038/etcd
tcp6       0      0 [::]:ssh                [::]:*                  LISTEN      1081/sshd
tcp6       0      0 [::]:4001               [::]:*                  LISTEN      3038/etcd
```

**Comment by mariussturm** [comment]
We saw the same issue for some other people as well. Unfortunately we were never able to reproduce this in our own environment. Is there any chance for us to get access to this box?

**Comment by zeacom** [comment]
Hi,

thank you for your answer. How can I send you the keys for access to the
server?

2015-03-06 10:16 GMT+01:00 Marius Sturm <notifications@github.com>:

**Comment by mariussturm** [comment]
Could you please put this key https://github.com/mariussturm.keys into /home/ubuntu/.ssh/authorized_keys and make sure port 22 is open in the security group? I just need the IP then.

**Comment by zeacom** [comment]
The current public IP is 54.93.182.250

2015-03-09 13:01 GMT+01:00 Marius Sturm <notifications@github.com>:

**Comment by mariussturm** [comment]
Thanks a lot!
In this case it is simple. Your instance type is too small, 1 Gig of ram is not enough. You can see it in the server log /var/log/graylog/server/current: "There is insufficient memory for the Java Runtime Environment to continue."

**Comment by zeacom** [comment]
Ok, makes sense. Now it works. Thx for your help!!!

2015-03-09 14:13 GMT+01:00 Marius Sturm <notifications@github.com>:

**Comment by magurwara** [comment]
Same problem.  I did the initial configuration of password for admin and timezone.  All services are running but cannot see anything on port 12900 and getting No Graylog Servers running... Checked server log /var/log/graylog/server/current and see "ERROR: org.graylog2.bootstrap.CmdLineTool - Invalid Configuraiton".

No idea why? Running in VMWare Workstation 7.1.3 on Windows 7 64-bit with 4GB allocated to the VM.
